examples/advanced/quantum_volume.py

```diff
@@ -104,6 +104,40 @@ def compute_heavy_set(circuit: cirq.Circuit) -> List[int]:
     ]
 
 
+def sample_heavy_set(circuit: cirq.Circuit,
+                     heavy_set: List[int],
+                     *,
+                     sampler: cirq.Sampler = cirq.Simulator()) -> float:
+    """Run a sampler over the given circuit and compute the percentage of its
+       outputs that are in the heavy set.
+
+    Args:
+        circuit: The circuit to sample.
+        heabv_set: The previously-computed heavy set for the given circuit.
+        sampler: The sampler to run on the given circuit.
+
+    Returns:
+        A probability percentage, from 0 to 1, representing how many of the
+        output bit-strings were in the heaby set.
+
+    """
+    # Add measure gates to the end of (a copy of) the circuit.
+    circuit_copy = circuit + [cirq.measure(*sorted(circuit.all_qubits()))]
+
+    # Run the sampler to compare each output against the Heavy Set.
+    repetitions = 1000
```

Comment:
this should likely be an argument to the function, perhaps with this default

Reply:
Done